src/cpu/o3/store_set.cc

```diff
@@ -177,6 +178,8 @@ StoreSet::violation(Addr store_PC, Addr load_PC)
         } else {
             SSIT[load_index] = store_SSID;
 
+            DPRINTF(ViolationCheck, "Load Index: %d, Store Index: %d", load_index, store_index);
```

Comment:
I think this is the case in violation() that we care about, but I am going to email the TA